### ISSUE TITLE
Feature Request -> Store Token in Local Storage

### ISSUE THREAD

**Issue by benhysell** [sounj142/HLSoft.BlazorWebAssembly.Authentication.OpenIdConnect#2]
It would be great to have the choice between session storage and local storage.  

An example config I use from TypeScript:
```
import { UserManagerSettings, WebStorageStateStore } from 'oidc-client';

config = {
                    authority: configuration.authority,
                    automaticSilentRenew: true,
                    client_id: configuration.spaClientId,
                    loadUserInfo: true,
                    post_logout_redirect_uri: configuration.postLogoutRedirectUri,
                    redirect_uri: configuration.spaRedirectUri,
                    response_type: 'token id_token',
                    scope: `openid profile email ${configuration.scope}`,
                    silent_redirect_uri: configuration.silentRedirectUri,
                    userStore: new WebStorageStateStore({ store: localStorage }),
                };
```

The key line being:
```
userStore: new WebStorageStateStore({ store: localStorage }),
```

This would allow the token to survive multiple tabs without requiring the user to log in if they open a new tab.

**Comment by sounj142** [comment]
Actually, at the beginning I was considering whether or not to implement
this configuration. The OAuth 2.0 best practices say that tokens should not be saved in Client Cookies / Local storage, but I think this is probably acceptable, some guys may like it. 
Currently I'm busy with examples for Google, Auth0.com, deal with end_session_endpoint issue ... I will come back to implement this function later, perhaps this weekend or next week.

**Comment by sounj142** [comment]
Hi,
I just released new version, 0.6.1 preview 2. You can add the code bellow to index.html to change the user storage:
```javascript
<script>
window.HLSoftBlazorWebAssemblyAuthenticationOpenIdConnect.configUserStore(window.localStorage);
</script>
```

**Comment by benhysell** [comment]
Works as advertised, thanks so much!
